src/components/nav-dropdown-button/styles.ts

```diff
@@ -0,0 +1,37 @@
+import { styled } from '@linaria/react'
+import { ElIcon } from '../icon'
+
+export const ElNavDropdownButton = styled.button`
+  cursor: pointer;
+  border: none !important;
+  height: 32px;
+  display: inline-flex;
+  padding: var(--spacing-1) var(--spacing-3);
+  gap: var(--spacing-1);
+  border-radius: var(--corner-default);
+  background-color: var(--fill-white);
+  color: var(--text-secondary);
+  align-items: center;
+  font-family: var(--font-family);
+  font-size: var(--font-size-sm);
+  font-weight: 500;
+  line-height: var(--line-height-sm);
+  letter-spacing: var(--letter-spacing-sm);
+
+  ${ElIcon} {
+    pointer-events: none;
+    margin-left: 0;
+  }
+
+  &:focus {
```

Comment:
**note:** we need to make sure we're consistent with how we apply focus styles. If we need to have a discussion as a team to decide between `:focus` and `:focus-visible`, let's have that discussion now via Teams/a GitHub discussion before we get too many components implemented in an inconsistent manner.

Reply:
or should we ask a design team, whether they want the `focus` style to be appear when user click the button 🤔

Reply:
seems like we can rever to original button component which user `:focus-visible` 

![image](https://github.com/user-attachments/assets/e316b0d8-cb6e-48fa-b33c-a4dae37aa298)